.github/workflows/swift_package_test.yml

```diff
@@ -27,6 +27,10 @@ on:
         type: string
         description: "Linux command to execute before building the Swift package"
         default: ""
+      musl_pre_build_command:
+        type: string
+        description: "Linux command to execute before building the Swift package"
```

Comment:
I think the description here just needs updated, as it's the same as the linux_pre_build_command